Cargo.toml

```diff
@@ -202,6 +202,8 @@ procinfo = { git = "https://github.com/tikv/procinfo-rs", rev = "6599eb9dca74229
 # After the PR to kvproto is merged, remember to comment this out and run `cargo update -p kvproto`.
 # [patch.'https://github.com/pingcap/kvproto']
 # kvproto = { git = "https://github.com/your_github_id/kvproto", branch = "your_branch" }
+[patch.'https://github.com/tikv/yatp.git']
```

Comment:
why change it？

Reply:
Temporary change. Make tikv be able to fetch min_thread_count and max_thread_count from ThreadPool.